nucypher/characters/lawful.py

```diff
@@ -1027,10 +1027,12 @@ def from_teacher_uri(cls,
                          min_stake: int,
                          network_middleware: RestMiddleware = None,
                          registry: BaseContractRegistry = None,
+                         retry_attempts: int = 2,
+                         retry_interval: int = 2
```

Comment:
Any reason for the retry interval changing from 10 to 2? Just to make it faster?

Also the retry attempts went from 3 to 2...?

Reply:
Yeah, just in an effort to increase(decrease..?) tolerance against failed connections to hardcoded nodes and take a little less time blocking.  I *think* we want the user to not really notice if the hardcoded seeds are unreachable, and be allowed to proceed unobstructed... We do want to raise to bar to entry too high by enforcing preferred nodes too strictly.

Additional thoughts?

Reply:
Makes sense - I guess testing will tell.